README.md

```diff
@@ -309,6 +317,20 @@ https://medium.com/matic-network/understanding-dagger-453d90480c51
   - One way to prevent DDoS is to check state validation when user submits the transaction (before tx goes to pending pool)
   - Node must handle state check validation part while accepting transactions. If any node decides to skip this implementation, other nodes will ignore all future transactions from particular node.
 
+17. Exit
+
+  - Q. Why Matic doesn't have Exit despite of it is like Plasma?
+  - A. Exit is not necessarily needed for sidechain. It is one of security model to achieve customer asset preservation. In our model, Delegates reselection ceremony and fast chain resuming are the security model for assets.
+
+18. Network layer censorship
+
+  - Q. Ethereum doesn't have network layer censorship-resistance for now, but bitcoin have. The reason why Ethereum doesn't have: Using UDP on network layer and cannot use Tor for hidden-mode. If geth apply libp2p, it is possible to be censorship-resistant. Then, how about Matic? Matic is using geth.
+  - A. Bitcoin and Ethereum have been regarded as commodity by U.S. SEC. Now it's lower possibility to censor comodity-cryptocurrency for now. In the future, depends on priority, Geth will implement it and we also apply it on Matic.
```

Comment:
We will change node for Matic. Currently, we are using Geth because we don't have resources to build node.

Reply:
> We go with Geth for 2~3yrs, Geth will be far improved until then. And we plan to be independent from Geth if there's inevitable restriction from Geth.

This kinda answer I will ready for.

Reply:
JD is right. We are currently using Geth at the Block Producer level because of lack of resources. Now since going forward that will not be the case, we intend to aggressively look for alternatives - probably Tendermint will be useful here or something comparable. Also, Geth wouldn't necessarily be optimum for fast transaction processing, especially at 1-second block times. We will need to look at other comparable implementations.

Regarding UDP and thereby Tor support, we will keep this mind for the node implementation. I don't necessarily like leaning on Geth support for libp2p for us to adopt it :)